ee/set_env_vars.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+echo "# needed for build/release/Makefile
+export DOCS_PREFIX=""
+export DOCS_GIT_REPO=git@github.com:koor-tech/docs.koor.tech.git
+export DOCKER_REGISTRY=koorinc
+export S3_BUCKET=koor.releases
+export HELM_BASE_URL=https://charts.koor.tech
+export HELM_S3_BUCKET=koor.chart" >> .env
```

Comment:
Using EOF is easier to read imo
```suggestion
cat <<EOF >> .env
# needed for build/release/Makefile
export DOCS_PREFIX=""
export DOCS_GIT_REPO=git@github.com:koor-tech/docs.koor.tech.git
export DOCKER_REGISTRY=koorinc
export S3_BUCKET=koor.releases
export HELM_BASE_URL=https://charts.koor.tech
export HELM_S3_BUCKET=koor.chart
EOF
```

Reply:
Also, where is this file used?

Reply:
this file is no longer needed, since we are using .env directly